src/data/community-meetups.json

```
@@ -346,5 +346,11 @@
     "emoji": ":cr:",
     "location": "Costa Rica",
     "link": "https://twitter.com/EthereumCR"
+  },
+  {
+    "title": "Hash House Web3 and Blockchain Community at Medellín",
```

```suggestion
    "title": "Hash House Medellín",
```
This should match the name of the event. Will update to this.